next.config.js

```diff
@@ -12,6 +12,8 @@ module.exports = withPlugins([[indexSearch], [feed], [sitemap]], {
 
   trailingSlash: true,
 
+  //verbose: true,
```

Comment:
would you mind adding a quick comment above this line for someone hopping in that might not necessarily be familiar with the verbose concept?

Reply:
Sure thing! Changing it

Reply:
perfect thanks